README.md

```diff
@@ -12,26 +12,23 @@ be banned from being added to the playlist/queue.
 
 ## Finished Features
 
+- Visually appealing landing Page
 - Firebase and Spotify authentication
 - Song controller ( shows current playing song on spotify and can pause/resume it )
 - Song selector ( allows users to search and select a song they want to battle for )
 - MiniGame ( tic tac toe as a way to battle for which song gets added )
-
-## Planned Features
-
-- Visually appealing landing Page
 - Spotify playlist/queue Component
 - MiniGame song handling
   - Adding winning song to playlist/queue
-  - Banning lost song from being added again for *x* seconds
+  - Banning lost song from being added until *party reset*
   - Save winning and losing song to state
 - Components for winner and loser song from state
 - Wiring component functionality (e.g. song selector reset when a game is won)
 - State / Persistence separation
 - Better API error handling
 - Styling
 
-### Low Priority Features
+### Possible Improvements
```

Comment:
I think these can just be left out

Reply:
Yep I would also leave them out, I think we have quite a lot of features already :)